target-app/Home/Views/HomeViewController.swift

```diff
@@ -6,13 +6,55 @@
 //
 
 import UIKit
+import MapKit
 
 class HomeViewController: UIViewController {
-
+    
+    let mapView: MKMapView = {
+        let map = MKMapView()
+        map.overrideUserInterfaceStyle = .dark
+        return map
+    }()
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
+        
         applyDefaultUIConfigs()
+        setupNavigationBar()
+        setupMapConstraints()
     }
-
+    
+    func setupNavigationBar() {
+        navigationController?.setNavigationBarHidden(false, animated: true)
```

Comment:
this should be done in viewDidAppear, so if you navigate to a different screen that hides the navigation bar, you make sure this screen will still show it after coming back.